android-test/src/androidTest/java/okhttp/android/test/OkHttpTest.kt

```diff
@@ -5,7 +5,7 @@
  * you may not use this file except in compliance with the License.
  * You may obtain a copy of the License at
  *
- *      http://www.apache.org/licenses/LICENSE-2.0
+ *      https://www.apache.org/licenses/LICENSE-2.0
```

Comment:
Can we leave these out.  The Apache license text is still http, and www.apache.org is probably in browsers HSTS preload files anyway.

Reply:
Reverted now.